common/changes/@rushstack/node-core-library/import-improvements_2022-12-09-01-39.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/node-core-library",
+      "comment": "Improve the error message emitted when a path inside a package is passed to `Import.ResolvePackage`.",
```

Comment:
Capitalization should be `resolvePackage`

Reply:
```suggestion
      "comment": "Improve the error message emitted when a path inside a package is passed to `Import.resolvePackage`.",
```